FrameworkSystem/Service/SystemAdministratorHandler.py

```diff
@@ -561,7 +563,7 @@ def __readHostInfo( self ):
       result['SecondsLeft'] = resultCert['Value']['secondsLeft']
       result['CertificateValidity'] = str( timedelta( seconds = resultCert['Value']['secondsLeft'] ) )
       result['CertificateDN'] = resultCert['Value']['subject']
-      result['HostProperties'] = ','.join( resultCert['Value']['groupProperties'] )
+      result['HostProperties'] = str( resultCert['Value']['groupProperties'] )
```

Comment:
resultCert['Value']['groupProperties'] is a string. join make no sense here and as well str. But I wanted to make sure that if it is a list it will work.